plan/physical_plan_test.go

```diff
@@ -407,6 +407,10 @@ func (s *testPlanSuite) TestDAGPlanBuilderJoin(c *C) {
 			sql:  "select /*+ TIDB_INLJ(t1) */ * from t t1 join t t2 where t1.f=t2.f and t1.a=t2.a",
 			best: "IndexJoin{TableReader(Table(t))->TableReader(Table(t))}(t1.a,t2.a)",
 		},
+		{
+			sql:  "select /*+ TIDB_INLJ(t1) */ * from t t1 join t t2 where t1.a=t2.a and t1.a in(1, 2) and t2.a in (1, 2)",
+			best: "IndexJoin{TableReader(Table(t))->TableReader(Table(t)->Sel([in(t2.a, 1, 2)]))}(t1.a,t2.a)",
+		},
```

Comment:
Seems like this test sql would not trigger the code path of this patch, i.e, `getIndexJoinByOuterIdx -> buildRangeForIndexJoin`, because column `a` is handle, it would follow `getIndexJoinByOuterIdx -> constructInnerTableScan`. Should we change the column `a` in the sql to column `b` instead? or am I missing anything?